notes/zrq/20200915-01-zeppelin-build.txt

```diff
@@ -207,3 +207,17 @@ EOF
             "/zeppelin/k8s/zeppelin-server.yaml"
 
 
+
+# -----------------------------------------------------
+# -----------------------------------------------------
+# Experiments with Angular/HTML.
```

Comment:
A bit confused here. This seems like standard html, is there a snippet missing with Angular code?

Reply:
Yep - description is not clear.
This is a HTML fragment that works in an Angular interpreter notebook cell.
This just experimenting with the HTML classes that fit into the CSS generated by the Zeppelin Angular UI.